### PR TITLE
 Add CSV export feature

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

fixes: https://github.com/goharbor/harbor/issues/17081

### CODE REVIEW

**Comment by a-mccarthy** [comment]
@AllForNothing If you have a minute, can you please review?

One question I have is the only way to download through the Event log window? It looks like once you click the download option, the CVS is removed.

**Comment by AllForNothing** [comment]
@a-mccarthy 
_is it the only way to download through the Event log window?_
It's the only way on UI. And you can also call the download API directly
_It looks like once you click the download option, the CVS is removed._
This is by design. (It helps save space in Harbor and improve performance)

**Comment by AllForNothing** [comment]
@a-mccarthy After the performance test, we found that exporting (all) multiple projects would cause performance issues.
So we decided to disable exporting (all) multiple projects.  And related PRs are Here:https://github.com/goharbor/harbor/pull/17265(for back-end),https://github.com/goharbor/harbor/pull/17267(for UI)
Once these PRs merged, I will build a new Harbor for you to update this DOC PR

**Comment by AllForNothing** [comment]
@a-mccarthy 
This env(https://10.202.250.189) has been upgraded to the latest version. And you can start to update this PR now.

Updates:
1. now, you can only export(select) one project at one time, and you can only click the `export button` once in 30 seconds
2. Add a new  type of permission for the export function, only `system admin, project admin, maintainer, developer` have this exporting permission:
image

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:75]: As a `project admin or maintainer or developer`
only able to export one project at one time

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:77]: at lease `developer`

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:79]: should remove this step

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:93]: formatted as YYYYMMDDHHMMSS.  It is now accurate to the second

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:96]: Once finishing download, the related execution record will be removed

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (AllForNothing) [inline docs/administration/vulnerability-scanning/scan-individual-artifact.md:84]: typo:  in the CSV, not CVS

**Comment by AllForNothing** [comment]
@a-mccarthy You can refer to this https://confluence.eng.vmware.com/display/HARBOR/Test+cases+for+CVE+data+export
to help you better understand the CVE export function

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (a-mccarthy) [reply docs/administration/vulnerability-scanning/scan-individual-artifact.md:83]: updated!

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (a-mccarthy) [reply docs/administration/vulnerability-scanning/scan-individual-artifact.md]: updated

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (a-mccarthy) [reply docs/administration/vulnerability-scanning/scan-individual-artifact.md]: removed

File: docs/administration/vulnerability-scanning/scan-individual-artifact.md
Comment (a-mccarthy) [reply docs/administration/vulnerability-scanning/scan-individual-artifact.md]: updated

**Comment by a-mccarthy** [comment]
@AllForNothing thanks for the updates! i think i've updated the docs to align with the new functionality. can you review again?

**Review by AllForNothing — APPROVED** [review]
LGTM

**Review by prahaladdarkin — APPROVED** [review]
looks good to me

**Review by chlins — APPROVED** [review]
lgtm

**Pull request merged**
